packager/react-packager/src/DependencyResolver/fastfs.js

```diff
@@ -297,7 +297,7 @@ class File {
     Object.keys(this.children).forEach(key => {
       const file = this.children[key];
       if (file.isDir) {
-        files.push(...file.getFiles());
+        file.getFiles().forEach((f) => files.push(f));
```

Comment:
FWIW, I just did a little test and indeed Node won't apply more than 65536 arguments to a function. And one of my `files.getFiles()` arrays is 122501 long.

So knowing this limit, we could also apply the array in chunks... Seems like a micro-optimization to me, but happy to do it if people feel it'd make a difference. Thoughts?